hack/e2e/run.sh

```diff
@@ -189,30 +197,37 @@ fi
 loudecho "Driver deployment complete, time used: $secondUsed seconds"
 
 loudecho "Testing focus ${GINKGO_FOCUS}"
-eval "EXPANDED_TEST_EXTRA_FLAGS=$TEST_EXTRA_FLAGS"
-set -x
-set +e
-${GINKGO_BIN} -p -nodes="${GINKGO_NODES}" -v --focus="${GINKGO_FOCUS}" --skip="${GINKGO_SKIP}" "${TEST_PATH}" -- -kubeconfig="${KUBECONFIG}" -report-dir="${ARTIFACTS}" -gce-zone="${FIRST_ZONE}" "${EXPANDED_TEST_EXTRA_FLAGS}"
-TEST_PASSED=$?
-set -e
-set +x
-loudecho "TEST_PASSED: ${TEST_PASSED}"
 
-OVERALL_TEST_PASSED="${TEST_PASSED}"
-if [[ "${EBS_CHECK_MIGRATION}" == true ]]; then
-  exec 5>&1
-  OUTPUT=$(ebs_check_migration "${KUBECONFIG}" | tee /dev/fd/5)
-  MIGRATION_PASSED=$(echo "${OUTPUT}" | tail -1)
-  loudecho "MIGRATION_PASSED: ${MIGRATION_PASSED}"
-  if [ "${TEST_PASSED}" == 0 ] && [ "${MIGRATION_PASSED}" == 0 ]; then
-    loudecho "Both test and migration passed"
-    OVERALL_TEST_PASSED=0
-  else
-    loudecho "One of test or migration failed"
-    OVERALL_TEST_PASSED=1
-  fi
+if [[ $TEST_PATH == "./tests/e2e-kubernetes/..." ]]; then
+  pushd $(pwd)/tests/e2e-kubernetes
+  packageVersion=$(echo $(cut -d '.' -f 1,2 <<< $KOPS_VERSION))
+
+  kubetest2 noop \
```

Comment:
don't we need `set -x` and `set +e` stuff here not to crash out on test failure (see next diff)

Reply:
Good eyes, added this 👍